server/test/e2e/asset.e2e-spec.ts

```diff
@@ -31,6 +32,9 @@ const user2Dto = {
   name: 'User 2',
 };
 
+const today = DateTime.fromObject({ year: 2023, month: 11, day: 3 });
```

Comment:
Immich' birthday? Please? :)

Reply:
feel free to open a PR to change it :-). We really need a constant for it, I can't be bothered to look it up all the time.

Reply:
Well now it's in the milestones at least. So easier to look up than scrolling through PRs